src/watch.py

```diff
@@ -0,0 +1,32 @@
+import time
+from pathlib import Path
+
+from watchdog.observers import Observer
+from watchdog.events import FileSystemEventHandler
+
+
+class Handler(FileSystemEventHandler):
+    @staticmethod
+    def on_any_event(event):
+        if event.is_directory:
+            return None
+        elif event.event_type == 'created':
+            return None
+        elif event.event_type == 'modified':
+            # TODO: Here we want to run the file tests instead of displaying its path
+            print("Received modified event - {}.".format(event.src_path))
+
+
+def watch(dir_path: Path):
```

Comment:
same here, encapsulate in a `FilesystemWatcher`. Will be easier to test!

Reply:
Done